### PR TITLE
Usagov 1547 explore redirect

### PR DESCRIPTION
**Pull request by YaritzaGarcia**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1547

## Description

I changed the redirect of usa.gov/explore from usa.gov/partner-with-usagov to our English homepage.

I added a line in the CMS configuration so it can be tested locally, but we **must remove it before merging**.

## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [x] Other

## Testing Instructions

1. Run "bin/build" and wait until it finishes.
2. Run "docker compose up".
3. Go to http://localhost/explore and check that it redirects to the homepage.
4. If all the links work correctly, write a comment on this PR saying that everything works but don't merge the changes until I remove the line in the CMS configuration.

### Requires New Config
- [x] Yes
- [ ] No

### Requires New Content
- [ ] Yes
- [x] No

### Validation Steps
- Verify that all links redirect correctly.

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
5. Find the commit of this pull request.
6. Build and deploy the changes.
7. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: .docker/src-cms/etc/nginx/partials/cms.conf.tmpl [hunk 1]
```diff
@@ -11,6 +11,9 @@ server {
     # CMS should not be indexed even if exposed.
     add_header X-Robots-Tag noindex;
 
+    # IT MUST BE REMOVED BEFORE MERGING INTO DEV!
```
Comment (akf) [inline .docker/src-cms/etc/nginx/partials/cms.conf.tmpl]: Remember to remove this.

**Comment by akf** [comment]
@YaritzaGarcia it works! Please remove that line from the cms configuration and tag me, and I'll approve and merge.

**Comment by YaritzaGarcia** [comment]
@akf Line, removed! Thank you

**Review by akf — APPROVED** [review]
Looks good and worked locally.

**Pull request merged**